docs/mac_development.md

```diff
@@ -44,7 +44,13 @@ podman-machine-default ssh://core@localhost:49473/run/user/501/podman/podman.soc
 ```
 example use the uid 501 above and port 49473 from your enviornment in the next step
 
-2. ssh -fnNT -L/tmp/podman.sock:/run/user/{uid}/podman/podman.sock -i ~/.ssh/podman-machine-default ssh://core@localhost:{port} -o StreamLocalBindUnlink=yes
+2. Create a secure tunnel, and forward the local socket to the remote socket 
+```
+RUN_USER_ID=501
+REMOTE_SSH_PORT=49473
```

Comment:
Why are these lines necessary? Isn't there an "export" missing in front or be present in the same line as the `ssh` command? people from @ansible/eda-maintainers with apple could verify this instructions.